### ISSUE TITLE
New GoPhish Instance Results in 400 Error

### ISSUE THREAD

**Issue by nategoethel** [gophish/gophish#2315]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.11.0

Brief description of the issue: Opening `localhost:3333` results in a 400 HTTP error.

What are you expecting to see happen? : The landing page appears.

What are you seeing happen? Nothing. Chrome shows its 400 error page.

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Run gophish from macOS terminal.
 * Step 2 - Open localhost:3333 in a browser. See error.

Here is my config.json:
```json
{
	"admin_server": {
		"listen_url": "127.0.0.1:3333",
		"use_tls": true,
		"cert_path": "gophish.crt",
		"key_path": "gophish.key"
	},
	"phish_server": {
		"listen_url": "0.0.0.0:80",
		"use_tls": false,
		"cert_path": "example.crt",
		"key_path": "example.key"
	},
	"db_name": "sqlite3",
	"db_path": "gophish.db",
	"migrations_prefix": "db/db_",
	"contact_address": "",
	"logging": {
		"filename": "",
		"level": ""
	}
}
```
 
Here is the request from Chrome:
```
Request URL: http://127.0.0.1:3333/
Request Method: GET
Status Code: 400 Bad Request
Remote Address: 127.0.0.1:3333
Referrer Policy: strict-origin-when-cross-origin
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9
Cache-Control: max-age=0
Connection: keep-alive
Host: 127.0.0.1:3333
sec-ch-ua: "Chromium";v="94", "Google Chrome";v="94", ";Not A Brand";v="99"
sec-ch-ua-mobile: ?0
sec-ch-ua-platform: "macOS"
Sec-Fetch-Dest: document
Sec-Fetch-Mode: navigate
Sec-Fetch-Site: none
Sec-Fetch-User: ?1
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36
```

**Comment by atheriqbal** [comment]
Hi, 
Is there anything in gophish log?

**Comment by nategoethel** [comment]
This is mysteriously working now. I did get a "Your connection is not secure" error before the page loaded this time, though.

**Comment by nategoethel** [comment]
I've found that entering `localhost:3333` results in the 400 error but using `https://127.0.0.1:3333` opens the landing page.

**Comment by atheriqbal** [comment]
Not sure about mac, but something to do with host file entry. 
https://127.0.0.1:3333  opens landing page or admin page?

**Comment by nategoethel** [comment]
Admin login page.

**Comment by atheriqbal** [comment]
So is your issue resolved or you want it to run on "localhost"? 127.0.0.1 is the same

**Comment by glennzw** [comment]
Seems this was resolved, feel free to re-open if not.
